internal/embed/templates/global.tmpl

```diff
@@ -321,6 +321,18 @@ function reject_rpki_invalid() {
   {{ end }}
 }
 
+function force_rpki_valid() {
+  {{ if .RPKIEnable }}
+  if (net.type = NET_IP4) then {
+    if (roa_check(rpki6, net, bgp_path.last_nonaggregated) != ROA_VALID) then _reject("RPKI invalid");
+  }
+
+  if (net.type = NET_IP6) then {
+    if (roa_check(rpki4, net, bgp_path.last_nonaggregated) != ROA_VALID) then _reject("RPKI invalid");
```

Comment:
See comment for line 327 https://github.com/natesales/pathvector/pull/66/files#diff-7b306c4d372ca850e06a9aa3865d86129f1f6cb83ca9db60087b9cdeabcfe750R327